async_test.go

```diff
@@ -196,3 +196,63 @@ func TestLongRunningAsyncQuery(t *testing.T) {
 		}
 	}
 }
+
+func runLongRunningAsyncQuery(ctx context.Context, t *testing.T) {
+	runDBTest(t, func(dbt *DBTest) {
```

Comment:
Not sure if this is the best construction. What do you think about such changes:
1. The whole test function should be put in `runDBTest` (as in all other testes).
2. Remove this function and use just `go dbt.mustQueryContext(ctx, "CALL SYSTEM$WAIT(100, 'SECONDS')")` in the test.
WDYT?

Reply:
Updated